tests/lib/snaps/test-snapd-ssh-keys/meta/snap.yaml

```diff
@@ -4,10 +4,6 @@ summary: Basic ssh-keys snap
 description: A basic snap which access to ssh
 
 apps:
-  ssh:
-    command: bin/ssh
-    plugs: [ssh-keys,network-control]
-  check:
-    command: bin/check
+  cmd:
```

Comment:
This can be called `sh` and be defined like this `command: ../../../bin/sh` -- for example here https://github.com/snapcore/snapd/blob/master/tests/lib/snaps/test-snapd-sh/meta/snap.yaml#L5

This way you can simply use it as a generic shell command pass-through.

Reply:
Nice, thanks